examples/general/abs_home_path/src/lib.rs

```diff
@@ -178,52 +124,94 @@ fn ui() {
         .run();
 }
 
+// Combined test for context allowances (build script and test context)
 #[test]
-fn build_script_allowance() {
+fn context_allowance() {
     use dylint_internal::CommandExt;
     use std::{
         io::{Write, stderr},
         path::Path,
         process::Command,
+        process::Output,
     };
 
-    // Skip test if repository is not stored in the user's home directory
+    struct TestCase {
+        manifest_path: &'static str,
+        context_name: &'static str,
```

Comment:
```suggestion
        context_name: &'static str,
        manifest_path: &'static str,
```
Let's please put `context_name` first since it's used first in the code below.